dockerscripts/docker-entrypoint.sh

```diff
@@ -24,15 +24,15 @@ fi
 
 ## Look for docker secrets in default documented location.
 docker_secrets_env() {
-    local MINIO_ACCESS_KEY_FILE="/run/secrets/access_key"
-    local MINIO_SECRET_KEY_FILE="/run/secrets/secret_key"
+    local ACCESS_KEY_FILE="/run/secrets/$MINIO_ACCESS_KEY_FILE"
+    local SECRET_KEY_FILE="/run/secrets/$MINIO_SECRET_KEY_FILE"
 
-    if [ -f $MINIO_ACCESS_KEY_FILE -a -f $MINIO_SECRET_KEY_FILE ]; then
-        if [ -f $MINIO_ACCESS_KEY_FILE ]; then
-            export MINIO_ACCESS_KEY="$(cat "$MINIO_ACCESS_KEY_FILE")"
+    if [ -f $ACCESS_KEY_FILE -a -f $SECRET_KEY_FILE ]; then
+        if [ -f $ACCESS_KEY_FILE ]; then
+            export MINIO_ACCESS_KEY="$(cat "$ACCESS_KEY_FILE")"
         fi
-        if [ -f $MINIO_SECRET_KEY_FILE ]; then
-            export MINIO_SECRET_KEY="$(cat "$MINIO_SECRET_KEY_FILE")"
+        if [ -f $SECRET_KEY_FILE ]; then
+            export MINIO_SECRET_KEY="$(cat "$SECRET_KEY_FILE")"
```

Comment:
The code here can be simplified. `[ -f $ACCESS_KEY_FILE -a -f $SECRET_KEY_FILE ]` already checks if both files exist, so the repeated checks inside the if block can be removed.

Reply:
Yes you're right! I've also noticed this. Thank you. But i didn't want to change too much in a single pull request. So i left the code style as it was to make a review easier.